kubernetes/hmda-platform/README.md

```diff
@@ -0,0 +1,30 @@
+Pre-requisites
+- [k8ssandra](https://k8ssandra.io/)
+- [Strimzi](https://strimzi.io/)
+
+Install
+- Secrets
+```
+kubectl create secret generic cassandra-credentials --from-literal=cassandra.username=  --from-literal=cassandra.password=
+kubectl create secret generic inst-postgres-credentials --from-literal=username= --from-literal=password= --from-literal=host= --from-literal=url="jdbc:postgresql://postgresql:5432/hmda?user= &password= &sslmode=false"
+```
+- Configmap
+```
+kubectl apply -f https://github.com/cfpb/hmda-platform/tree/master/kubernetes/config-maps
+```
+
+Update
+```
+helm upgrade --install --force --namespace=default --values=kubernetes/hmda-platform/values-dev.yaml \
+--set image.tag=latest \
+--set service.name=hmda-platform \
+hmda-platform kubernetes/hmda-platform
+```
+
+Delete
+```
+kubectl delete secret cassandra-credentials
+kubectl delete secret generic inst-postgres-credentials
```

Comment:
deletion don't have `generic` keyword

Reply:
not sure if it's github, but I'm still seeing `generic` in this secret deletion.